corehq/apps/app_manager/static/app_manager/json/template_apps/case_management.json

```diff
@@ -80,9 +66,24 @@
       "logenabled": "Enabled"
     }
   },
+  "amplifies_workers": "not_set",
+  "comment_from": null,
+  "cloudcare_enabled": false,
+  "description": null,
+  "recipients": "",
+  "use_grid_menus": false,
+  "translations": {
+    
+  },
+  "built_on": null,
+  "deployment_date": null,
+  "archived_media": {
+    
+  },
   "langs": [
     "en"
   ],
+  "migrating_blobs_from_couch": true,
```

Comment:
Not sure if this should be included in the json since it's a static property on the class.